apps/web/src/layouts/default-layout.tsx

```diff
@@ -29,7 +30,8 @@ export default function DefaultLayout({
     <>
       <HeadTag pageDescription={pageDescription} pageTitle={pageTitle} />
       <div className={common.backgroundGradient}>
-        <EmailVerfication />
+        <EmailVerification />
+        <UntransferredPacks />
```

Comment:
May need to make these mutually exclusive.

Reply:
Or perhaps not? You need to have a verified email address before you can purchase a pack, right? @dallashuggins / @jakerainis

Reply:
Ah yes, that is correct, email verification is required to purchase a pack. So I don't think there'd be a case where someone would get both alerts.